include/capstone/m68k.h

```diff
@@ -78,7 +78,7 @@ typedef enum m68k_reg {
 
 //> M68K Addressing Modes
 typedef enum m68k_adress_mode {
-	M68K_AM_NONE = 0,			// No address mode. 
+	M68K_AM_NONE = 0,			// No address mode.
```

Comment:
i didnt pay attention to this before. for consistency, would you mind renaming this to M68K_AM_INVALID?